src/head.py

```diff
@@ -0,0 +1,35 @@
+def wrt(f, n, b):
```

Comment:
Can you add how to use that script in the comment? Put the command line you use yourself.